src/services/authentication/hooks/index.js

```diff
@@ -1,5 +1,9 @@
 const { TooManyRequests } = require('@feathersjs/errors');
 const { discard } = require('feathers-hooks-common');
+const jwt = require('jsonwebtoken');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['jwt' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4141092)